language/stdlib/modules/Roles.move

```diff
@@ -398,7 +398,7 @@ module Roles {
             except new_designated_dealer_role, grant_role;
         apply AbortsIfNotTreasuryCompliance{account: creating_account} to new_designated_dealer_role;
 
-        /// ParentVASP roles are only granted by LibraRoot [[A6]][ROLE]. A new `RoleId` with `PARENT_VASP_ROLE_ID()` is only
+        /// ParentVASP roles are only granted by TreasuryCompliance [[A6]][ROLE]. A new `RoleId` with `PARENT_VASP_ROLE_ID()` is only
```

Comment:
Good call! The governance requirement for this rule changed from LibraRoot to TreasuryCompliance. So, the code and spec were updated accordingly at that time, but it seems this comment was not.